hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/cli/TestCLI.java

```diff
@@ -43,7 +43,6 @@ public void tearDown() throws Exception {
   @Override
   protected CommandExecutor.Result execute(CLICommand cmd) throws Exception {
     return cmd.getExecutor("", conf).executeCommand(cmd.getCmd());
-
```

Comment:
The changes in Debian are as expected, and we can roll back the modifications to this unit as they are unrelated to our main changes.